examples/send-multiple-l2-tx/main.go

```diff
@@ -0,0 +1,140 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"math/big"
+
+	"github.com/hermeznetwork/hermez-go-sdk/account"
+	"github.com/hermeznetwork/hermez-go-sdk/client"
+	"github.com/hermeznetwork/hermez-go-sdk/node"
+	"github.com/hermeznetwork/hermez-go-sdk/transaction"
+	hezCommon "github.com/hermeznetwork/hermez-node/common"
+)
+
+const (
+	ethereumNodeURL           = ""
+	chainID                   = 5 // set this accordingly to the ethereum node
+	auctionContractAddressHex = "0x1D5c3Dd2003118743D596D7DB7EA07de6C90fB20"
```

Comment:
As far as we try to send L2 txs, why is the smart contract address needed??

Reply:
Because it needs to find the actual coordinator.